File: src/main/java/com/m2t/hachikodictionary/model/Quote.java
package com.m2t.hachikodictionary.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Quote {
    private String quote;
    private String author;
    private String difficulty;

}

File: src/main/java/com/m2t/hachikodictionary/model/Quote.kt
package com.m2t.hachikodictionary.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.Id
import jakarta.persistence.Table
import org.hibernate.annotations.GenericGenerator


@Entity
@Table(name = "quotes")
data class Quote(
    @Id
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator") // Generate UUID
    val id: String,

    @Column(unique = true)
    val quote: String,
    val author: String,
    val difficulty: String
) {
    constructor(quote: String, author: String, difficulty: String) : this(
        id = "",
        quote = quote,
        author = author,
        difficulty = difficulty
    ) {

    }
}